proto/internal/temporal/server/api/historyservice/v1/service.proto

```diff
@@ -306,6 +306,10 @@ service HistoryService {
     rpc TrimHistoryBranch(TrimHistoryBranchRequest) returns (TrimHistoryBranchResponse) {
     }
 
+    // DeleteHistoryBranch deletes history branch
```

Comment:
I think we should remove this comment because it looks like we don't enforce comments and this one doesn't provide anything meaningful.